.travis.yml

```diff
@@ -87,7 +88,13 @@ jobs:
   - stage: special
     os: linux
     compiler: clang
-    env: CC=clang-9 CXX=clang++-9 CMAKE_BUILD_TYPE=Debug GTSAM_BUILD_UNSTABLE=OFF GTSAM_ALLOW_DEPRECATED_SINCE_V4=ON
+    env: CC=clang-9 CXX=clang++-9 CMAKE_BUILD_TYPE=Debug GTSAM_BUILD_UNSTABLE=OFF GTSAM_ALLOW_DEPRECATED_SINCE_V4=ON GTSAM_WITH_TBB=OFF
+    script: bash .travis.sh -b
+# on Linux, with GTSAM_WITH_TBB not off to make sure GTSAM still compiles/tests
+  - stage: special
```

Comment:
Wouldn't it make more sense to use the special stage to run with TBB ON on OSX and Linux for both Release and Debug builds? TBB is optional after all.

Reply:
I was going off @dellaert comments [here]()

Reply:
Cool!